bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -260,4 +260,14 @@ public Map<String, Object> getDorProperties() {
         }
         return customDorProperties;
     }
+
+    @Override
+    @JsonIgnore
+    public String getParentPagePath() {
+        String path = this.getPath();
+        if (!StringUtils.isEmpty(path)) {
```

Comment:
i'd prefer if we implement it by going up the hierarchy till we find a cq:Page node and then return its path.
locating jcr:content in path seems hackish.

Reply:
Hope there won't be a performance impact in highly nested hierarchy seen in most sites.

Reply:
Done!